README.md

```diff
@@ -108,6 +108,16 @@ the `-p` option (long form `--probe`), which can have the following values:
   (e.g., `usb-0`, `usb-1`, etc.)  To determine which probe is which,
   examine the serial number in the output of `humility probe`.
 
+- `vid:pid[:serial]`: In some cases, the automatic algorithm may either find
+  the wrong thing, or timeout attempting to search for non-existent probes.
+  In these cases, it can help to explicitly set the vendor ID (VID) and
+  product ID (PID) of the debug probe, which should be colon-delimited, e.g.,
+  `0483:374e`.  (Determining the VID and PID of an attached probe is
+  platform-specific; on Linux one can use the `lsusb` command.)  In cases
+  where there are multiple probes with the same VID and PID, the serial number
+  of the probe (as reported via `humility probe` can be postpended, also
```

Comment:
I'd like to somehow indicate that this serial number is also the USB device serial number. Maybe something like 'the USB Device Serial Number (as reported by `humilty probe` or platform-specific tools like `libusb`)'. I dunno. Maybe you can phrase that in a way that sounds better. There's also a missing closing paren ')' after humility probe in this line.

Reply:
Fixed -- thank you!